.envExample

```diff
@@ -8,5 +8,6 @@ FILE_SIZE_UPLOAD_LIMIT_MB=512
 BLOCKED_MIME_TYPES=application/x-dosexec,application/x-executable,application/x-hdf5,application/x-java-archive,application/vnd.rar
 USE_CLOUDFLARE=false
 MAX_URL_LENGTH=4096
-RECAPTCHA_SITE_KEY=Put-Your-Google-V2-Recaptcha-Site-Key-Here
-RECAPTCHA_SECRET_KEY=Put-Your-Google-V2-Recaptcha-Secret-Key-Here
\ No newline at end of file
+CAPTCHA_SERVICE=reCAPTCHA
+CAPTCHA_SITE_KEY=Put-Your-Google-V2-Recaptcha-Site-Key-Here
+CAPTCHA_SECRET_KEY=Put-Your-Google-V2-Recaptcha-Secret-Key-Here
```

Comment:
we should remove thius, captcha is not required and this makes it seem that it needs to be google.

Reply:
also use `#` for comment in .env

Reply:
Removed